src/elements/__tests__/transform.spec.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  transfromElementIn,
```

Comment:
```suggestion
  transformElementIn,
```

...etc 😁

Reply:
Took me quite a while to spot the difference there!